### PR TITLE
Acquire Rstudio instance

### PR DESCRIPTION
**Pull request by thomasyu888**
* Uses ami-0226a8af83fcecb43 found from here: http://www.louisaslett.com/RStudio_AMI/
* Wait until auto encryption of instances is pulled in for this PR to be pulled in
* Need this instance for analysis of GENIE data

### CODE REVIEW

**Comment by zaro0508** [comment]
auto encryption of instances is in PR https://github.com/Sage-Bionetworks/aws-infra/pull/150

File: config/prod/tom-rstudio.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "{{stack_group_config.default_vpc}}"
```
Comment (zaro0508) [inline config/prod/tom-rstudio.yaml]: Should be ”computevpc”

File: config/prod/tom-rstudio.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  KeyName: "{{stack_group_config.default_key_pair}}"
```
Comment (zaro0508) [inline config/prod/tom-rstudio.yaml]: Should be “scicomp”

File: config/prod/tom-rstudio.yaml (hunk 1 above)
Comment (thomasyu888) [reply config/prod/tom-rstudio.yaml]: Why the change?   Are you getting rid of `stack_group_config`?

File: config/prod/tom-rstudio.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "computevpc"
+  KeyName: "scicomp"
+  AMIId: "ami-0226a8af83fcecb43"
```
Comment (zaro0508) [inline config/prod/tom-rstudio.yaml]: [Optional] would be helpful to add a comment telling people where this AMI came from.

**Pull request merged**